.github/workflows/continuous-integration-e2e.yaml

```diff
@@ -65,6 +65,10 @@ jobs:
         CHAIN_HISTORY_PROVIDER: 'http'
         CHAIN_HISTORY_PROVIDER_PARAMS: '{"baseUrl":"http://localhost:4000/chain-history"}'
         DB_SYNC_CONNECTION_STRING: 'postgresql://postgres:doNoUseThisSecret!@localhost:5435/cexplorer'
+        HANDLE_PROVIDER: 'http'
+        HANDLE_PROVIDER_PARAMS: '{"baseUrl":"http://localhost:4000/handle","policyId":""}'
+        KORA_LABS_HANDLE_PROVIDER: 'kora-labs'
+        KORA_LABS_HANDLE_PROVIDER_PARAMS: '{"serverUrl":"http://localhost:4000","policyId":""}'
```

Comment:
Why is the serverUrl from localhost? Shouldn't it be the public API?

Reply:
updated in [09a16b9](https://github.com/input-output-hk/cardano-js-sdk/pull/791/commits/09a16b91052a9842893bd7d2db6c72a9aea040d2)